app/services/foreman_register/registration_token.rb

```diff
@@ -11,20 +14,20 @@ def encode(host, secret)
       private
 
       def prepare_payload(host, secret)
-        exp = iat + (24 * 3600)
-        nbf = iat - 3600
-        jti_raw = [secret, iat].join(':')
+        expire_at = issued_at + EXPIRE_AFTER_ISSUING
+        not_before = issued_at - VALID_BEFORE_ISSUING
+        jti_raw = [secret, issued_at].join(':')
```

Comment:
If you decided to "unpack" those shortcuts, why you haven't unpacked `jti` as well? :)